Dockerfile

```diff
@@ -13,6 +17,12 @@ RUN bundle install --jobs 4 && npm install && npm install -g phantomjs-prebuilt
 
 EXPOSE 3000
 ADD . /myapp
-RUN RAILS_ENV=production bundle exec rake assets:precompile
+RUN if [ -z "$CUSTOM_ASSETS_URL" ]; then\
```

Comment:
This should be extracted to a bash script that will be run on CircleCI - download the assets onto the host, put them into a directory where they will be reached by `rake assets:precompile` - e.g. `/public`, and keep the Dockerfile as it was.